doc/changelog/02-specification-language/13312-attributes+bool_single.rst

```diff
@@ -0,0 +1,17 @@
+- **Changed:**
+  :term:`Boolean attributes <boolean attribute>` are now specified using
+  key/value pairs, that is to say :n:`@ident__attr{? = {| on | off } }`.
+  If the value is missing, the default is :n:`on`.  The old syntax is still
+  supported, but produces the ``deprecated-attribute-syntax`` warning.
+
+  Deprecated attributes are :attr:`universes(monomorphic)`,
```

Comment:
I don't get why we need to deprecate the current attributes.
My reading wast that the boolean value `true` was implicit.

Reply:
Your reading is correct. Only the negative attributes are deprecated. Note that while `monomorphic` is a fine name, `notemplate` and `noncumulative` not so much.

Reply:
Could we undeprecate monomorphic then?

Reply:
I guess we could, although this would require special code.

>`monomorphic` is a fine name

To contradict myself a little, `monomorphic` can be a bit misleading since these two things are accepted:

```coq
#[ universes(monomorphic, template) ] Inductive foo.

#[ universes(monomorphic, notemplate) ] Inductive bar.
```

while only one of them is truly monomorphic. Anyway, I'm not opposed to keeping `monomorphic` forever if you wish.

Reply:
I think we should deprecate things which are dangerous or block cleanups. Uniformity is to be performed by tools imo, not humans.

Sure, having 3 ways to turn univ polymorphism off may be unneeded, but as a user I would not be happy to be warned to do such a gratuitous change (`monomorphic` -> `polymoprhic=off`).

Reply:
I'm a bit lost here, isn't it what we are doing a case of "uniformity" being enforced by tools?